tests/specs/examples/erc20-bin-runtime.k.expected

```diff
@@ -0,0 +1,90 @@
+requires "edsl.md"
+
+module ERC20-BIN-RUNTIME
+    imports public EDSL
+    
+    syntax Bool ::= "unknownBool" [klabel(unknownBool), symbol(), no-evaluators(), function(), functional()]
+    
+    syntax ERC20Contract ::= "ERC20" [klabel(contract_ERC20)]
+    
+    syntax ERC20Function ::= "allowance" "(" Int "," Int ")" 
+    
+    syntax ERC20Function ::= "approve" "(" Int "," Int ")" 
+    
+    syntax ERC20Function ::= "balanceOf" "(" Int ")" 
+    
+    syntax ERC20Function ::= "decimals" "(" ")" 
+    
+    syntax ERC20Function ::= "name" "(" ")" 
+    
+    syntax ERC20Function ::= "symbol" "(" ")" 
+    
+    syntax ERC20Function ::= "totalSupply" "(" ")" 
+    
+    syntax ERC20Function ::= "transfer" "(" Int "," Int ")" 
+    
+    syntax ERC20Function ::= "transferFrom" "(" Int "," Int "," Int ")" 
+    
+    syntax ByteArray ::= ERC20Contract "." ERC20Function [klabel(function_ERC20), macro()]
+    
+    rule  ( ERC20.allowance(OWNER, SPENDER) => #abiCallData("allowance", #address(OWNER), #address(SPENDER)) )
+      
+       ensures ( #rangeAddress(OWNER)
+       andBool ( #rangeAddress(SPENDER)
+               ))
```

Comment:
I don't think this can be a `macro`, the `ensures` clause won't be passed through correctly.

You can either make it a `function` instead, or you can inpsect the generated Kore for a claim that uses this and see if it passes through the ensures clause correctly.